src/modules/SideBarDrawer/HelpDrawer.js

```diff
@@ -36,7 +44,7 @@ const HelpDrawer = ({helpText, skillInfo, skillsOnly, onClose, classes}) => {
             <CloseIcon />
           </IconButton>
         </Grid>
-        {!skillsOnly && (
+        {!skillsOnly && isOpenDrawer1 && (
```

Comment:
I think you need to wrap the next `<Grid>` item (on line 76) in a `isOpenDrawer1` block as well, right?